app/mailers/purchase_notifier.rb

```diff
@@ -0,0 +1,54 @@
+class PurchaseNotifier < ActionMailer::Base
+
+  include DateHelper
+  add_template_helper ApplicationHelper
+  add_template_helper TranslationHelper
+  add_template_helper OrdersHelper
+  add_template_helper ViewHookHelper
+
+  default from: Settings.email.from, content_type: "multipart/alternative"
+
+  # Notifies the specified recipient if an order is placed including a product
+  def product_order_notification(order_detail, recipient)
+    @order = order_detail.order
+    @order_detail = OrderDetailPresenter.new(order_detail)
+    attach_reservation_ical(order_detail.reservation) if order_detail.reservation.present?
+    send_nucore_mail recipient, text("views.purchase_notifier.product_order_notification.subject", product: order_detail.product)
+  end
+
+  def order_notification(order, recipient)
```

Comment:
Can we add a comment on this method on what it's doing that's different from product_order_notification?